crates/ruff_python_formatter/src/statement/stmt_import_from.rs

```diff
@@ -32,6 +35,11 @@ impl FormatNodeRule<StmtImportFrom> for FormatStmtImportFrom {
                 space(),
             ]
         )?;
+
+        let comments = f.context().comments().clone();
+        let dangling_comments = comments.dangling_comments(item.as_any_node_ref());
+        write!(f, [trailing_comments(dangling_comments)])?;
+
```

Comment:
I would have expected the comments to be written right before the `names` to match the logic in `placement.rs`: The dangling comments are dangling comments after any possible opening parentheses, meaning they should come after the `*`.